internal/rpc/clients.go

```diff
@@ -33,6 +35,8 @@ var (
 		"app":       "openmatch",
 		"component": "client",
 	})
+	pool *x509.CertPool
```

Comment:
This is a legit lint finding. Having a global pool is bad for a few reasons.
Unit tests are now going to read global state so this will be a source of flakes and race conditions.

I'd much prefer reading the certs each time and caching the objects. We can improve that later. If we have a global that's much harder to change.